### ISSUE TITLE
attribute in JSON payload with boolean value false turns into undefined

### ISSUE THREAD

**Issue by moehlone** [balderdashy/sails.io.js#44]
Steps to reproduce:

(1) Send request with JSON payload to any route (I used .post() method in this case) and add a param with boolean value false -> e.g. {"some_flag":false}
(2) log your param on server side
e.g. console.log(req.param('some_flag'));
(3) note that logged value is typeof undefined instead of boolean (false)

Expected: boolean value false

Sending values with true works as expected.

Best regards,
Phil

**Comment by sgress454** [comment]
Thanks for the report @moehlone, we'll take a look!

**Comment by sgress454** [comment]
I am unable to reproduce this with the latest version of Sails and sails.io.js.  Can you confirm that you're using Sails v0.11.0?

**Comment by moehlone** [comment]
Yes, using sails 0.11.0 and sails.io.js 0.11.5 - tested with a new generated app.

**Comment by moehlone** [comment]
I repeated my steps again and reproduced this issue. In a few hours I will try on another system with a fresh sails npm installation. Thank you for checking!

**Comment by sailsbot** [comment]
Thanks for posting, @moehlone.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
